src/cript/api/utils/save_helper.py

```diff
@@ -53,3 +120,18 @@ def find_node_by_uuid(node, uuid: str):
     missing_node = node.find_children({"uuid": uuid})[0]
 
     return missing_node
+
+
+def _identify_suppress_attributes(node, response: Dict) -> Dict[str, Set[str]]:
+    suppress_attributes = {}
+    if response["error"].startswith("Additional properties are not allowed"):
+        # Find all the attributes, that are listed in the error message with regex
+        attributes = set(re.findall(r"'(.*?)'", response["error"]))
+        path = response["error"][response["error"].rfind("path:") + len("path:") :].strip()
```

Comment:
what does this intend to do?

Reply:
I will explain in the comments.
We are parsing the error message string to get the info we need.